### ISSUE TITLE
Docs build is failing

### ISSUE THREAD

**Issue by chinmayshah99** [OpenMined/PyDP#303]
## How to Reproduce
https://github.com/OpenMined/PyDP/runs/1097568856

**Comment by BenjaminDev** [comment]
I believe the issue is in build_PyDP.sh:
```
#!/bin/bash
pipenv install --dev --skip-lock
bazel coverage src/python:bindings_test --verbose_failures # THIS LINE needs to specify the config to use 
find ./ -name _pydp.so -print0 | xargs -0 -I {} rm {}
cp -f ./bazel-bin/src/bindings/_pydp.so ./pydp
```
So we need to update it to:
```
#!/bin/bash
pipenv install --dev --skip-lock
bazel build --config Linux src/python:bindings_test  --verbose_failures
find ./ -name _pydp.so -print0 | xargs -0 -I {} rm {}
cp -f ./bazel-bin/src/bindings/_pydp.so ./pydp
```

**Comment by BrendanSchell** [comment]
@chinmayshah99 this can be closed now I think since it should be resolved by #306
